libnetwork/drivers/bridge/bridge_linux.go

```diff
@@ -1261,33 +1252,10 @@ func (d *driver) CreateEndpoint(ctx context.Context, nid, eid string, ifInfo dri
 	if err = d.linkUp(ctx, host); err != nil {
 		return fmt.Errorf("could not set link up for host interface %s: %v", hostIfName, err)
 	}
-
-	// Generate a MAC-address-based IPv6 address, which may be used by the default
-	// bridge network instead of an IPAM allocated address.
-	if endpoint.addrv6 == nil && config.EnableIPv6 {
-		var ip6 net.IP
-		network := n.bridge.bridgeIPv6
-		if config.AddressIPv6 != nil {
-			network = config.AddressIPv6
-		}
-		// If there are fewer than 48 host-bits in the network, it's not possible to
-		// generate an IPv6 address from the MAC address. The network size has already
-		// been checked, so we can safely assume this endpoint must not need a MAC-based
-		// IPv6 address.
-		ones, _ := network.Mask.Size()
-		if ones <= 80 {
-			ip6 = make(net.IP, len(network.IP))
-			copy(ip6, network.IP)
-			for i, h := range endpoint.macAddress {
-				ip6[i+10] = h
-			}
-
-			endpoint.addrv6 = &net.IPNet{IP: ip6, Mask: network.Mask}
-			if err = ifInfo.SetIPAddress(endpoint.addrv6); err != nil {
-				return err
-			}
-		}
-	}
+	log.G(ctx).WithFields(log.Fields{
+		"hostifname": host.Attrs().Name,
+		"ifi":        host.Attrs().Index,
+	}).Debugf("bridge endpoint host link is up")
```

Comment:
nit: don't bother if not doing other changes

```suggestion
	}).Debug("bridge endpoint host link is up")
```

Reply:
Done.